src/renderers/webgpu/utils/WebGPUBindingUtils.js

```diff
@@ -373,8 +373,29 @@ class WebGPUBindingUtils {
 
 					const usage = GPUBufferUsage.UNIFORM | GPUBufferUsage.COPY_DST;
 
+					const visibilities = [];
+					if ( binding.visibility & 1 ) {
+
+						visibilities.push( 'vertex' );
+
+					}
+
+					if ( binding.visibility & 2 ) {
+
+						visibilities.push( 'fragment' );
+
+					}
+
+					if ( binding.visibility & 4 ) {
+
+						visibilities.push( 'compute' );
+
+					}
+
+					const bufferVisibility = `(${visibilities.join( ',' )})`;
+
 					const bufferGPU = device.createBuffer( {
-						label: 'bindingBuffer_' + binding.name,
+						label: `bindingBuffer${binding.id}_${binding.name}_${bufferVisibility}`,
```

Comment:
Um, we should not overload the label with information. Why you have added the ID?